PCbuild/pyproject.props

```diff
@@ -42,7 +42,7 @@
 
   <PropertyGroup>
     <_DebugPreprocessorDefinition>NDEBUG;</_DebugPreprocessorDefinition>
-    <_DebugPreprocessorDefinition Condition="$(Configuration) == 'Debug'">_DEBUG;</_DebugPreprocessorDefinition>
+    <_DebugPreprocessorDefinition Condition="$(Configuration) == 'Debug'">_DEBUG;Py_DEBUG;</_DebugPreprocessorDefinition>
```

Comment:
Definitely don't need or want this here - it's in the header files.

Reply:
We shouldn't even need the `NDEBUG` or `_DEBUG` variables here, as they should be set by the compiler. But best to just leave this alone entirely - there's no need to change it in this PR.

Reply:
Reverted.